src/core_plugins/console/public/src/directives/settings.html

```diff
@@ -1,67 +1,91 @@
-<div class="kuiLocalDropdownTitle">Settings</div>
+<h2 class="kuiLocalDropdownTitle">Settings</h2>
 
 <form class="form" name="settingsForm" ng-submit="settingsForm.$valid && settings.apply()">
-  <div class="form-group">
+  <fieldset
+    class="kuiVerticalRhythm"
+    role="group"
+  >
     <div class="kuiLocalDropdownHeader">
-      <div class="kuiLocalDropdownHeader__label">Font Size</div>
-    </div>
-
-    <div>
-      <input
-        input-focus
-        ng-model="settings.vals.fontSize"
-        name="fontSize"
-        type="number"
-        required
-        class="form-control"
-        data-test-subj="setting-font-size-input"
+      <legend
+        id="consoleFontSize"
+        class="kuiLocalDropdownHeader__label"
       >
+        Font Size
+      </legend>
     </div>
 
-    <div class="checkbox">
-      <label>
-        <input type="checkbox" ng-model="settings.vals.wrapMode"/> Wrap long lines
-      </label>
-    </div>
+    <input
+      input-focus
+      ng-model="settings.vals.fontSize"
+      name="fontSize"
+      type="number"
+      required
+      class="kuiLocalDropdownInput kuiVerticalRhythmSmall"
+      aria-describedby="consoleFontSize"
```

Comment:
I think we should rather use `aria-labelledby` here, so that "Font Size" will really become the *accessible name* of that input field.